festim/exports/surface_quantity.py

```diff
@@ -27,6 +27,10 @@ def __init__(self, field, surface, filename: str = None) -> None:
         self.data = []
         self._first_time_export = True
 
+    @property
+    def title(self):
+        return f"{self.field.name} surface {self.surface.id}"
+
```

Comment:
The function `title` is called in the `write` function further in the file, but since it's its only occurence it may be better to replace it by something else ?

Reply:
Well the thing is that this title is meaningless really. No one will ever just create a `SurfaceQuantity`, do nothing with it, and then call `compute` (because `compute` doesn't exist!). The only use for this class is to bundle the shared functionality between classes like `MaximumSurface` `SurfaceFlux` etc.

Really the only use for this class is to be inherited by another class that will define the `compute` method and the `title` property.

We should simply remove this property `title` at the `SurfaceQuantity` and `VolumeQuantity` level